wait-for-it.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
Better to use the old one in case `bash` is installed elsewhere (or a newer version is in one's `PATH`.